### DISCUSSION TITLE
Poll: What coefficient should we use to increase ASA Stats Community Rewards ?

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#5] [Polls]
The ASA Stats Community Rewards, seen here under [Rewards Model tab](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233), have diminished in value as the ASASTATS token price has fluctuated over the past 6 months.  Based on the current valuations in the spreadsheet, 1hr of content work (editing or generation) is valued at .01 damo (10,000 ASASTATS) - which equates to roughly ~3.00 USD.

The purpose of this poll is to ask the Community what they feel is an appropriate adjustment coefficient for rewards in this modeled in this spreadsheet.  By using [CT] as a reference, we hope to apply a flat adjustment to all rewards classifications.

**Comment by Zamanikings** [comment]
I think x2 works... And average of $6/hour is a good start.... Depending on the growth and value of tthe ASASTATS TOKEN... SUBSEQUENT INCREASES CAN BE MADE

**Comment by moloch10** [comment]
What was the multiplier recently used?

**Reply by AlgoRhythMatic** [reply]
We used a 2x multiplier when averaging from 1/01/22 -> 3/25/22.

**Comment by AlgoRhythMatic** [comment]
Thanks to those who voted!  Looks like a 2x multiplier is the most favored coefficient, so that is what we'll use for the current cycle/s.  We'll revisit this in the future as valuation fluctuates!
